config/application.yml.default

```diff
@@ -50,6 +50,7 @@ doc_auth_extend_timeout_by_minutes: '40'
 document_capture_step_enabled: 'false'
 document_capture_react_enabled: 'true'
 email_from: no-reply@login.gov
+email_from_display_name: Login.gov
```

Comment:
A lot of our user-facing copy goes into our translations files, I'm double checking with our design team in case we want to translate this in different languages -- if so, we should move it out of this config file

Reply:
From our translation team:

> would leave it as "Login.gov", everybody knows they need to login or sign on, or sign in, English is ubiquitous in the digital space and I feel folks understand, even if they choose to use a French or Spanish alternative.

Reply:
Confirmed, no change needed to this part https://github.com/18F/identity-idp/pull/4058#issuecomment-673683562